src/features/intl-displaynames.md

```diff
@@ -0,0 +1,112 @@
+---
+title: '`Intl.DisplayNames`'
+author: 'Shu-yu Guo ([@_shu](https://twitter.com/_shu)) and Frank Tang'
+avatars:
+  - 'shu-yu-guo'
+  - 'frank-tang'
+date: 2020-02-13
+tags:
+  - Intl
+  - Node.js 14
```

Comment:
I don't think this is a valid tag (with spaces), and on the previous [Intl post](https://v8.dev/features/intl-numberformat) we just included `Intl` - let's do the same here?

Reply:
It's a valid tag (search for `Node.js 12`, for example). But generally we only add these tags for Node.js versions that have been released already.

Reply:
If you open any of the posts containing that tag in the source, you can actually see that such tags with a space also seem to have silently broken the parser and both them and any following tags are not rendered on the pages.

For example: https://github.com/v8/v8.dev/blob/f880711d0ebd70413e3a3cecb1e7f3dbaaa969ab/src/features/intl-relativetimeformat.md#L7-L10

contains `Intl`, `Node.js 12` and `io19` in the source, but

https://v8.dev/features/intl-relativetimeformat

renders only `Intl`.

Reply:
This is not a bug, in fact we have special logic just to achieve this: https://github.com/v8/v8.dev/blob/b93cbfc168b4b19ecebf5a8c93ada7ec8cf3bdd2/src/_includes/features.njk#L7 We don't want #io and Node.js tags to show up on the overview, but we do want to be able to link to them.

Reply:
That... wasn't very obvious, but okay :)